### PR TITLE
Sanitize tracer output

### LINKED ISSUE #85: capture function arguments instead of entire stack

### PR DESCRIPTION
**Pull request by pik694**
Return only relevant elems of stack

Closes #85

### CODE REVIEW

File: src/instrumentation_measurement/measurements.py [hunk 1]
```diff
@@ -159,6 +165,45 @@ def csv_row_append_info(self, instrumenter_result, program, sample_id):
...
+        for i in range(len(args), MAX_OPCODE_ARGS):
+            args.append('')
+
+        parsed_line = ','.join(prefix + args)
+        result.append(parsed_line)
+
+      return result
+
+  def read_opcodes_specs(self):
+      with open('./instrumentation_measurement/data/opcodes_args.csv', newline='') as opcodes_args_file:
```
Comment (pdobacz) [inline src/instrumentation_measurement/measurements.py:193]: probably safer to use path relevant to this file, see `program_generator.py` for example

Comment (pdobacz) [reply src/instrumentation_measurement/measurements.py:193]: 2nd comment: can't we just derive the information we need from the `opcodes.csv` file we have? Arity of the OPCODEs should be enough, except SWAP and DUP.

I'm afraid these files would be cumbersome to keep in sync

**Review by pdobacz — COMMENTED** [review]
can't we derive the required data from the existing CSV file? see comment

File: src/instrumentation_measurement/measurements.py (hunk 1 above)
Comment (pik694) [reply src/instrumentation_measurement/measurements.py:193]: I guess that it might be a good idea - I can merge it :)

**Review by pdobacz — APPROVED** [review]
Thanks!

**Pull request merged**